src/SonarScanner.MSBuild.PreProcessor/PreprocessorObjectFactory.cs

```diff
@@ -80,6 +80,8 @@ internal static HttpClient CreateHttpClient(string userName, string password, st
             }
 
             var client =  new HttpClient(handler);
+            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(new ProductHeaderValue("SonarScanner-for-.NET", Utilities.ScannerVersion)));
```

Comment:
I'm intentionally using a different value than the old one:
* We are rebranding scanner from `MSBuild` to `.NET`
* Different value will not break anybody, because nobody uses the new header (yet)
* We will avoid breaking users in the future